### DISCUSSION TITLE
Cannot rebind "TAB" key to apply suggestion in VS 2022

### DISCUSSION THREAD

**Discussion by fbdesignpro** [community/community#120851] [Copilot Conversations]
### Select Topic Area

Bug

### Body

There is no way to use any other key then TAB in Visual Studio 2022 (C++) to apply a copilot suggestion.

It seems that this behavior can be customized in VS Code, but not Visual Studio. I tried many suggestions in other forums but none work. There are already options for modifying shortcuts for all sorts of behavior such as "Trigger suggestion", but nothing to use any other key than TAB to accept the suggestion.

This is a deal-killer for using Copilot personally, as I get SO MANY inadvertent suggestions applied the TAB key is used as a matter of course during programming.

When you rollover the suggestion, you get a "[TAB] to accept [Gear icon]" tooltip, suggesting clicking on the gear would enable us to change that behavior. However the tooltip goes away and it is impossible to click the gear icon, so I assume this is a bug.

**Comment by ayim** [comment]
**Hello from Visual Studio team, before I propose a solution, what is the ideal key binding you'd want to use to accept your code?**
Tracking feedback and status in this item here: https://developercommunity.visualstudio.com/t/GitHub-Copilot-Completions---cannot-rebi/10688822

_Potential Solution_: there's an option to use Right Arrow to accept your code prediction.
You can do this by changing the setting in the IntelliCode component that Copilot uses to display and commit predictions.

![image](https://github.com/community/community/assets/10951869/5d40bc03-9969-43ee-9f8d-9de6ae88ddd6)
![image](https://github.com/community/community/assets/10951869/3c128627-9d08-4c53-8f55-750ec43eb85d)
